packages/yew/src/app.rs

```diff
@@ -31,31 +31,34 @@ where
     /// which will update the state of the model and a `view` function which
     /// will render the model to a virtual DOM tree. If you would like to pass props,
     /// use the `mount_with_props` method.
-    pub fn mount(self, element: Element) -> ComponentLink<COMP> {
+    pub fn mount(mut self, element: Element) -> Self {
```

Comment:
I don't think these methods should return the `App`. Doesn't it makes more sense to use `&mut self`?

Reply:
> Doesn't it makes more sense to use &mut self?
Yes it does make more sense. I think I did it like like that because I wanted to minimize the fn signature diff, but it doesn't really make sense when it's a breaking change anyway.

Reply:
Wait a minute... I also did it because the `Scope::mount_in:place` takes `self`.
So either I can just work around that by creating a temporary empty `Scope`, or I can change the signature of `Scope::mount_in:place` (from what I can see, it could just as well take  `&self`).